src/instructlab/sdg/datamixing.py

```diff
@@ -397,9 +424,17 @@ def __init__(self, output_dir, date_suffix, sys_prompt, num_procs):
         self.date_suffix = date_suffix
         self.num_procs = num_procs
 
-        self.knowledge_recipe = Recipe(sys_prompt=self.sys_prompt)
-        self.skills_recipe = Recipe(sys_prompt=self.sys_prompt)
+        self.knowledge_recipe = Recipe(
+            recipe_path="src/instructlab/sdg/configs/knowledge/data_recipe/default_recipe.yaml",
```

Comment:
Hmm, this is relative to CWD? How do we know the CWD is `instructlab/sdg` ? What when `ilab` is consuming the library as a wheel?

For loading pipeline configs, we load it from the Python package: https://github.com/instructlab/sdg/blob/cbee53c2453876f2602c86ef3e5a803689604716/src/instructlab/sdg/generate_data.py#L229

Reply:
That's a great catch on the CWD, as I know I've only tested from within an instructlab/sdg clone folder and I suspect that's how others are testing, which is why this happens to work as-is but would not reliably work. I do think we should be able to load and reference these files like the configs instead of the more naive way here.

Reply:
Resolving this, as the code has evolved significantly from here and addresses this concern.